Task1.PianoKeyboard/Enums/ENote.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Task1.PianoKeyboard.Enums
+{
+    internal enum ENote
```

Comment:
Do not use "E" prefix, it is not common practice, https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/names-of-classes-structs-and-interfaces